tracer/src/Datadog.Trace.Trimming/build/Datadog.Trace.Trimming.xml

```diff
@@ -546,6 +548,8 @@
       <type fullname="System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute" />
       <type fullname="System.Diagnostics.CodeAnalysis.NotNullWhenAttribute" />
       <type fullname="System.Diagnostics.ConditionalAttribute" />
+      <type fullname="System.Diagnostics.Debug" />
+      <type fullname="System.Diagnostics.Debug/AssertInterpolatedStringHandler" />
```

Comment:
```suggestion
```

These are added when a `debug` build is made, but we actually don't want them to be committed to this file.